tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,372 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "T_eff = 4780 * u.K  # temperature of hatp11 [K]\n",
+    "\n",
+    "# Use synphot.models and synphot.spectrum module to generate a blackbody spectrum\n",
+    "photlam_hatp11 = SourceSpectrum(BlackBody1D, temperature=T_eff) # in photon s-1 cm^-2 AA^-1\n",
```

Comment:
The comment `in photon s-1 cm^-2 AA^-1` is misleading here. At this point, `photlam_hatp11` is an instance of `SourceSpectrum`, which _contains_ the `BlackBody1D` model describing the star. As far as user is concerned, flux unit does not come into play until you sample the spectrum.

In addition, usually when creating a theoretical source spectrum, one normalizes it to some reference flux in a reference filter. For example, I can create a blackbody with this temperature, but then I normalize the blackbody to have the property of, say, 28 ABmag when observed in, say, *V*-band (this depends on existing observational data of your object of interest). A source spectrum without normalization like this sometimes give crazy results.

See also: [synphot.spectrum.BaseSourceSpectrum.normalize](https://synphot.readthedocs.io/en/latest/api/synphot.spectrum.BaseSourceSpectrum.html#synphot.spectrum.BaseSourceSpectrum.normalize)

Reply:
I did see that you normalized your spectra in the examples, but it's unclear to me why this is necessary. Is it just for comparison's sake?

Reply:
Normalization is needed when the default model flux values are arbitrary. For example, blackbody flux at a given temperature is the theoretical blackbody from Planck's law, which does not often correspond to real life observations. Say, for the Sun, you would want to normalized that blackbody at solar temperature to the observed solar magnitude at a given filter. Does this make sense?

Reply:
Maybe @eteq or @bmorris3 have the official academic explanation.

Reply:
I could be really wrong here, but my current understanding is that the generic blackbody flux in the `BlackBody1D` model should be the flux _at the surface_ of the source. So you need to normalize it by the square of the radius over the distance to the target in order to get the flux at Earth. Does that sound reasonable @pllim?

Reply:
Right, I thought scaling by r/d^2 would cover the normalization. As I mentioned somewhere below it might be good to use the normalization function for the second example to show both ways?

Reply:
Yeah, maybe what you did is already correct. I am not sure coz the count rate looks suspiciously high to me. Or maybe this is just a super bright star, even for APO. 🤷‍♀

Reply:
Anyways, sorry for the confusion. I think first of all, we need to check if count rate is correct. Then if not, we can revisit how the normalization is done.